packages/flame/test/components/component_test.dart

```diff
@@ -200,6 +200,56 @@ void main() {
       );
     });
   });
+
+  group('Component descendants', () {
```

Comment:
@Dan-Crane can we add a test where `descendants` is called before being added?

Reply:
yes, it will be returned 0 elements. I will add

Reply:
done